src/components/card/video-footer/VideoFooter.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classes from './VideoFooter.module.css';
+
+const VideoFooter = ({
+  title = 'Video Title',
+  creator = 'Amar',
+  views = '2',
+  date = '3',
+}) => {
+  return (
+    <div className={classes.footer}>
+      <img
+        class="avatar avatar-sm"
+        src="https://bit.ly/dan-abramov"
+        alt="medium logo"
+      />
+      <div className={classes['video-info']}>
+        <p className={classes.title}>{title}</p>
+        <p className={classes.creator}>{creator}</p>
+        <p className={classes.info}>
+          <span className={classes.views}>{views}k views - </span>
+          <span className={classes.date}>{date}years ago</span>
+        </p>
+      </div>
+      <span className={classes.menu}>
+        OPT
```

Comment:
You can give option icon here instead of OPT and also give a little more padding to the search bar. 
 The Ui looks clean and the code structure is good.

Reply:
Yes i will be putting a icon there. I will look into the padding of search bar. Thank you for the review.